roles/pulp_database_config/tasks/main.yml

```diff
@@ -34,7 +29,8 @@
     - name: Set the Pulp admin user's password
       command: '{{ pulp_django_admin_path }} reset-admin-password --password {{ pulp_default_admin_password }}'
       no_log: true
-      when: pulp_default_admin_password is defined and migrate_auth.changed
+      when: (pulpcore_install.changed and not pulp_pip_editable) or
```

Comment:
I believe the changed is True when we install or upgrade pulpcore, so if we do a z-stream release, do we want this task to run?

Reply:
@fao89 No, but we still have [#7499 open](https://pulp.plan.io/issues/7499) to get the "when" to our liking in the comint days. The current approach is good enough IMHO, and can be merged as soon as I verify the output.

Reply:
since we have another issue to track it, I agree this PR is good enough